.cspell.json

```diff
@@ -1,157 +1,156 @@
-        "ccid",
-        "celo",
-        "cids",
-        "clsx",
-        "commenceack",
-        "configtx",
-        "Corda",
-        "Cordapp",
-        "couchdb",
-        "COUCHDBADDRESS",
-        "COUCHDBCONFIG",
-        "data",
-        "dclm",
-        "DHTAPI",
-        "DockerOde",
-        "ealen",
-        "ecparams",
-        "embeddable",
-        "Errorf",
-        "escc",
-        "execa",
-        "faio",
-        "fidm",
-        "flowdb",
-        "fsouza",
-        "GETHKEYCHAINPASSWORD",
-        "ghcr",
-        "gopath",
-        "goquorum",
-        "grpc",
-        "grpcs",
-        "grpcwebtext",
-        "guks",
-        "hada",
-        "hashicorp",
-        "Healthcheck",
-        "htlc",
-        "Htlc",
-        "HTLC",
-        "HyperLedger",
-        "ipaddress",
-        "ipfs",
-        "Iroha",
-        "Irohad",
-        "isready",
-        "jboss",
-        "JORDI",
-        "jsrsa",
-        "jsrsasign",
-        "Keychain",
-        "Keycloak",
-        "KEYUTIL",
-        "KJUR",
-        "Knetic",
-        "LEDGERBLOCKACK",
-        "leveldb",
-        "lmify",
-        "LOCALMSPID",
-        "miekg",
-        "mitchellh",
-        "MSPCONFIGPATH",
-        "MSPID",
-        "MSPIDSCOPEALLFORTX",
-        "MSPIDSCOPEANYFORTX",
-        "Mtls",
-        "myapp",
-        "mychannel",
-        "myroot",
-        "mysecretpassword",
-        "myvolume",
-        "Nerc",
-        "NETWORKSCOPEALLFORTX",
-        "NETWORKSCOPEANYFORTX",
-        "NODETXPOOLACK",
-        "notok",
-        "Odap",
-        "Oidc",
-        "oneofs",
-        "onsi",
-        "OpenAPI",
-        "openethereum",
-        "organisation",
-        "Orgs",
-        "parameterizable",
-        "Postgres",
-        "proto",
-        "protobuf",
-        "protoc",
-        "protos",
-        "qscc",
-        "recoverupdateackmessage",
-        "RUSTC",
-        "Rwset",
-        "sbjpubkey",
-        "Secp",
-        "shrn",
-        "Smonitor",
-        "socketio",
-        "SPDX",
-        "Splug",
-        "Sprintf",
-        "stretchr",
-        "supervisorctl",
-        "supervisord",
-        "svcs",
-        "sykesm",
-        "tezos",
-        "TEZOS",
-        "thream",
-        "tlsca",
-        "tlscacerts",
-        "txid",
-        "txqueue",
-        "Uisrs",
-        "undici",
-        "Unmarshal",
-        "uuidv",
-        "vscc",
-        "wasm",
-        "Xdai"
-    ],
-    "dictionaries": [
-        "typescript,node,npm,go,rust"
-    ],
-    "ignorePaths": [
-        "**/node_modules/**",
-        "**/build/**",
-        "**/src/main/typescript/generated/**",
-        "packages/cactus-plugin-verifier-cc/**",
-        "packages/cactus-cmd-socketio-server/**",
-        "packages/cactus-plugin-ledger-connector-sawtooth-socketio/**",
-        "packages/cactus-plugin-ledger-connector-go-ethereum-socketio/**",
-        "packages/cactus-plugin-ledger-connector-*-socketio/**"
-    ]
+  "version": "0.1",
```

Comment:
@barnapa Please eliminate whitespaces from the diff.

Reply:
@petermetz Done in my latest force push. But please double check if it is what you wanted. :)

Reply:
@rwat17 The whitespace is still in the diff, attaching screenshot from my local checkout.


![Screenshot from 2023-05-25 17-07-44](https://github.com/hyperledger/cacti/assets/1583231/42a0e1c7-ae58-4527-a963-41acef80aab3)